Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest-tasks-navigation/ContestTasksNavigation.module.scss

```diff
@@ -5,14 +5,14 @@
 }
 
 .tasksListSideNavigation {
-  align-items: start;
+  align-items: flex-start;
   padding: 20px;
   display: flex;
 
   .taskSideNavigationItem {
     text-align: start;
     font-size: 18px;
-    font-family: 'Lato', serif;
+    font-family: Lato, serif;
```

Comment:
We can export this into a global variable and replace it everywhere

Reply:
https://github.com/SoftUni-Internal/exam-systems-issues/issues/225